### PR TITLE
Forward root mail

### PR DESCRIPTION
**Pull request by cwickert**
### Description

I've migrated the content of the old SmartDoc to the new template. Feel free to finish and fine-tune.

### CODE REVIEW

File: tasks/forward-root-mail.xml [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+        </para>
+      </step>
+    </procedure>
+  </section>
+  <section xml:id="task-forward-root-mail-summary">
+    <title>Summary</title>
+    <para>
+      You have now configured mail forwarding for the &rootuser; user.
+    </para>
+  </section>
```
Comment (dariavladykina) [inline tasks/forward-root-mail.xml:139]:
```suggestion
 
```

File: articles/forward-root-mail.asm.xml [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    </resource>
+    <resource href="../common/license_gfdl1.2.xml" xml:id="_gfdl">
+      <description>GNU Free Documentation License</description>
+    </resource>
+  </resources>
+  <!-- S T R U C T U R E -->
+  <structure renderas="article" xml:id="forward-root-mail" xml:lang="en">
+    <merge>
+      <title>Configuring mail forwarding for &rootuser;</title>
+      <subtitle>Subtitle if necessary</subtitle>
```
Comment (dariavladykina) [inline articles/forward-root-mail.asm.xml:52]:
```suggestion
  
```

File: articles/forward-root-mail.asm.xml [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+            </listitem>
+            </varlistentry> -->
+      </variablelist>
+      </abstract>
+    </merge>
+    <!-- pull in all the topic files you need -->
+    <!-- pick the appropriate type of include to match your needs -->
+    <!-- pull in a topic as is -->
+    <module resourceref="task-forward-root-mail" renderas="section"/>
+    <module resourceref="forward-root-mail-more-info" renderas="section"/>
```
Comment (dariavladykina) [inline articles/forward-root-mail.asm.xml:152]:
```suggestion
  
```

File: articles/forward-root-mail.asm.xml [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+        <dm:editurl>https://github.com/SUSE/doc-modular/tree/main/articles/</dm:editurl>
+        <dm:translation>no</dm:translation>
+      </dm:docmanager>
+      <abstract>
+      <variablelist>
+        <varlistentry>
+          <term>WHAT</term>
+          <listitem>
+            <para>
+              This article describes how to congiure mail forwarding for the &rootuser; user.
```
Comment (dariavladykina) [inline articles/forward-root-mail.asm.xml:117]:
```suggestion
              This article describes how to configure mail forwarding for the &rootuser; user.
```

File: glues/forward-root-mail-whats-next.xml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+ xmlns:xlink="http://www.w3.org/1999/xlink"
+ xmlns:trans="http://docbook.org/ns/transclusion">
+  <title>What's next?</title>
+  <info>
+    <!--add author's email address-->
+    <meta name="maintainer" content="cwickert@suse.com" its:translate="no"/>
+  </info>
+  <para>
+    Depending on the number of managed machines and the number of persons who
+    need to be informed about system events, different email address models can
```
Comment (dariavladykina) [inline glues/forward-root-mail-whats-next.xml:21]:
```suggestion
    need to be informed about system events, different e-mail address models can
```

File: glues/forward-root-mail-whats-next.xml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    be established:
+  </para>
+  <itemizedlist>
+    <listitem>
+      <para>
+        Collect messages from different systems in an email account that is only
```
Comment (dariavladykina) [inline glues/forward-root-mail-whats-next.xml:27]:
```suggestion
        Collect messages from different systems in an e-mail account that is only
```

File: glues/forward-root-mail-whats-next.xml [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+        accessed by a single person.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Collect messages from different systems in a group email account (aliases
```
Comment (dariavladykina) [inline glues/forward-root-mail-whats-next.xml:33]:
```suggestion
        Collect messages from different systems in a group e-mail account (aliases
```

File: glues/forward-root-mail-whats-next.xml [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+        or mailing list) that can be accessed by all relevant persons.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Create separate email accounts for each system.
```
Comment (dariavladykina) [inline glues/forward-root-mail-whats-next.xml:39]:
```suggestion
        Create separate e-mail accounts for each system.
```

File: glues/forward-root-mail-whats-next.xml [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+      </para>
+    </listitem>
+  </itemizedlist>
+  <para>
+    It is crucial that administrators regularly check the related email accounts.
```
Comment (dariavladykina) [inline glues/forward-root-mail-whats-next.xml:44]:
```suggestion
    It is crucial that administrators regularly check the related e-mail accounts.
```

File: tasks/forward-root-mail.xml [hunk 10]
```diff
@@ -0,0 +1,202 @@
...
+      System daemons, <systemitem class="daemon">cron</systemitem> jobs, &systemd;
+      timers, and other applications can generate messages and send them to the
+      &rootuser; user of the system. By default, each user account owns a local
+      mailbox and will be notified about new mail messages upon login.
+    </para>
+    <para>
+      These messages can contain security-relevant reports and incidents that
+      might require a quick response by the system administrator. To get notified
+      about these messages in a timely fashion, it is strongly recommended to
+      forward these mails to a dedicated remote email account that is regularly
```
Comment (dariavladykina) [inline tasks/forward-root-mail.xml:35]:
```suggestion
      forward these mails to a dedicated remote e-mail account that is regularly
```

File: tasks/forward-root-mail.xml [hunk 11]
```diff
@@ -0,0 +1,202 @@
...
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          The system is connected to the network.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          You have a remote email account that reliably receives messages.
```
Comment (dariavladykina) [inline tasks/forward-root-mail.xml:54]:
```suggestion
          You have a remote e-mail account that reliably receives messages.
```

File: tasks/forward-root-mail.xml [hunk 12]
```diff
@@ -0,0 +1,202 @@
...
+        </para>
+        <!-- <screen>&prompt.root;<command>zypper in yast2-mail</command></screen> -->
+      </listitem>
+    </itemizedlist>
+  </section>
+  <section xml:id="task-forward-root-mail-executing">
+    <title>Configure &rootuser; mail forwarding with &yast;</title>
+    <para>
+      The following procedure describes how to configure mail forwarding for the &rootuser; user to
+      a dedicated remote email account.
```
Comment (dariavladykina) [inline tasks/forward-root-mail.xml:70]:
```suggestion
      a dedicated remote e-mail account.
```

File: tasks/forward-root-mail.xml [hunk 13]
```diff
@@ -0,0 +1,202 @@
...
+          Send a message to test whether the mail forwarding works correctly:
+        </para>
+<screen>&prompt.user;<command>mail root</command>
+  subject: test
+  test
+  .</screen>
+      </step>
+      <step>
+        <para>
+          Check the remote email account for new messages. Your test message should
```
Comment (dariavladykina) [inline tasks/forward-root-mail.xml:128]:
```suggestion
          Check the remote e-mail account for new messages. Your test message should
```

File: tasks/forward-root-mail.xml [hunk 14]
```diff
@@ -0,0 +1,202 @@
...
+            Use the <command>mailq</command> command to verify that the test message
+            has been sent. Upon success, the queue should be empty.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Is the outgoing mail server resolved correctly?</term>
+        <listitem>
+          <para>
+            Check that the system resolves the mail server of the remote email account
```
Comment (dariavladykina) [inline tasks/forward-root-mail.xml:171]:
```suggestion
            Check that the system resolves the mail server of the remote e-mail account
```

File: tasks/forward-root-mail.xml [hunk 15]
```diff
@@ -0,0 +1,202 @@
...
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Does the test message still not arrive?</term>
+        <listitem>
+          <para>
+            If <command>mailq</command> reports the message has been sent
+            successfully, the problem must be on the receiving end. Check the
+            configuration of the remote email account and its spam folder.
```
Comment (dariavladykina) [inline tasks/forward-root-mail.xml:196]:
```suggestion
            configuration of the remote e-mail account and its spam folder.
```

File: tasks/forward-root-mail.xml [hunk 16]
```diff
@@ -0,0 +1,202 @@
...
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Can you reach the outgoing mail server?</term>
+        <listitem>
+          <para>
+            Verify you can ping the mail server with <command>ping -c 5
+              <replaceable>&mailname;</replaceable></command> (where
+            <replaceable>&mailname;</replaceable> is the host name of the mail server
+            returned by the <command>host</command> command in the previous step.)
```
Comment (dariavladykina) [inline tasks/forward-root-mail.xml:186]:
```suggestion
            returned by the <command>host</command> command in the previous step).
```

File: tasks/forward-root-mail.xml [hunk 17]
```diff
@@ -0,0 +1,202 @@
...
+      </varlistentry>
+      <varlistentry>
+        <term>Is the outgoing mail server resolved correctly?</term>
+        <listitem>
+          <para>
+            Check that the system resolves the mail server of the remote email account
+            correctly. Run <command>host -t mx
+              <replaceable>&exampledomain;</replaceable></command> (where
+            <replaceable>&exampledomain;</replaceable> is the is the domain of the
+            email account configured in step 5.)
```
Comment (dariavladykina) [inline tasks/forward-root-mail.xml:175]:
```suggestion
            <replaceable>&exampledomain;</replaceable> is the domain of the
            e-mail account configured in step 5).
```

**Review by dariavladykina — COMMENTED** [review]
Hi Christoph, please see some suggestions (I've omitted the Summary, For more info and subtitle)

File: tasks/forward-root-mail.xml (hunk 1 above)
Comment (cwickert) [reply tasks/forward-root-mail.xml:139]: No, that does not fly, we cannot just remove a section that is part of the template.

Comment (dariavladykina) [reply tasks/forward-root-mail.xml:139]: I've consulted @janajaeger and found out that we can do so.

**Pull request merged**